src/sass/components/button.scss

```diff
@@ -168,10 +275,24 @@
   @include tetra-button(tertiary);
 }
 
+.btn.btn-default-alt {
```

Comment:
Don't need to be nested

Should be only `.btn-default-alt {...}`, `.btn-tertiary-alt {...}`...

The common